x-pack/legacy/plugins/siem/server/lib/timeline/routes/import_timelines_route.ts

```diff
@@ -65,28 +67,34 @@ export const importTimelinesRoute = (
       },
     },
     async (context, request, response) => {
-      const siemResponse = buildSiemResponse(response);
-      const savedObjectsClient = context.core.savedObjects.client;
-      if (!savedObjectsClient) {
-        return siemResponse.error({ statusCode: 404 });
-      }
-      const { filename } = request.body.file.hapi;
+      try {
+        const siemResponse = buildSiemResponse(response);
+        const savedObjectsClient = context.core.savedObjects.client;
+        if (!savedObjectsClient) {
+          return siemResponse.error({ statusCode: 404 });
+        }
 
-      const fileExtension = extname(filename).toLowerCase();
+        const { file } = pipe(
```

Comment:
@angorayc this should be unnecessary, as the request will already have been decoded via `buildRouteValidation` above.